commands/create.go

```diff
@@ -61,30 +61,15 @@ func runCreate(dockerCli command.Cli, in createOptions, args []string) error {
 		}
 	}
 
-	buildkitHost := os.Getenv("BUILDKIT_HOST")
-
-	driverName := in.driver
-	if driverName == "" {
-		if len(args) == 0 && buildkitHost != "" {
-			driverName = "remote"
-		} else {
-			var arg string
-			if len(args) > 0 {
-				arg = args[0]
-			}
-			f, err := driver.GetDefaultFactory(ctx, arg, dockerCli.Client(), true)
-			if err != nil {
-				return err
-			}
-			if f == nil {
-				return errors.Errorf("no valid drivers found")
-			}
-			driverName = f.Name()
-		}
+	contexts, err := dockerCli.ContextStore().List()
+	if err != nil {
+		return err
 	}
-
-	if driver.GetFactory(driverName, true) == nil {
-		return errors.Errorf("failed to find driver %q", in.driver)
+	for _, c := range contexts {
+		if c.Name == in.name {
+			logrus.Warnf("instance name %q already exists as context builder", in.name)
+			break
+		}
```

Comment:
this should only be checked on creation. you can still leave or append a context builder to an existing one.